src/character.cpp

```diff
@@ -3468,6 +3470,14 @@ void Character::calc_discomfort()
             add_effect( effect_chafing, 1_turns, bp, true, 1 );
         }
     }
+    // do the same for obesity conflicting with rigid armors
+    remove_effect( effect_squeezed );
```

Comment:
I'd suggest adding a second flag for armor made by the character or maybe a hidden trait that tracks BMI for when the character received it and whether it fit then?  I'm trying not to overcomplicate it but it would be bizarre for the PC to received the `squeezed` status from armor they just finished making or that Cody made specially fitted to them.

Reply:
My thinking initially was that Cody was specifically making armor around your "healthy" weight and might refuse to even measure you if you were currently too overweight, and it does cause some questions about whether the armor should fit you when you're also thin, but I can go ahead and add a simple flag that makes it ignore BMI "BMI_BESPOKE" or something

Reply:
For now we're going to get rid of this check, and later on we will have clothing sized per different body sizes as a granular system. So no need to have this basic incomplete system in the meantime. McDonalds and platemail will remain compatible for the forseeable future.